### PR TITLE
Promote Mensa TUI

### PR DESCRIPTION
**Pull request by LukasPietzschmann**
As not everybody likes to use websites for literally everything, it would be nice if you could promote other ways of accessing the university's meal plan.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,9 @@ where YYYY-MM-DD is today's date.
 - General refactoring of JavaScript code
 - Changed URL Query string parsing to be order independent
 
+## Alternatives
```
Comment (Tanikai) [inline README.md:102]: Please add spacing after headings for better readability

**Review by Tanikai — CHANGES_REQUESTED** [review]
see comment

File: README.md (hunk 1 above)
Comment (LukasPietzschmann) [reply README.md:102]: Sure, did it :)

**Comment by Tanikai** [comment]
ty for contribution

**Pull request merged**